roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -47,3 +47,26 @@ openshift_logging_mux_ops_ca: /etc/fluent/keys/ca
 #mux_config_contents:
 #mux_throttle_contents:
 #mux_secureforward_contents:
+
+# Default value of mux BUFFER_TYPE
+openshift_logging_mux_buffer_type: "file"
+
+# One of ['emptydir', 'pvc', 'hostmount']
+openshift_logging_mux_file_buffer_storage_type: "emptydir"
+
+# mount path: shared by hostmount and pvc
+openshift_logging_mux_file_buffer_mount_path: /mux/filebufferstorage
+
+# pvc options
+# the name of the PVC we will bind to -- create it if it does not exist
+openshift_logging_mux_file_buffer_pvc_name: "mux-file-buffer-pvc"
```

Comment:
We should consider matching this to the DC name in such a way that it is the same pattern you get from a statefulset.  This would ensure in future that if we could or need to move mux to a statefulset that we do not need to worry about migrating the volume or PVC